app/browser/windows.js

```diff
@@ -62,51 +61,44 @@ const updateWindow = (windowId) => {
   }
 }
 
+const siteMatchesTab = (site, tab) => {
+  const matchesLocation = getLocationIfPDF(tab.get('url')) === site.get('location')
+  const matchesPartition = (tab.get('partitionNumber') || 0) === (site.get('partitionNumber') || 0)
```

Comment:
you could use Immutable's notSetValue parameter instead of `||`. https://facebook.github.io/immutable-js/docs/#/Map/get